From ddccac45ed3bd5ff7988d75e16e56ab39b71160a Mon Sep 17 00:00:00 2001
From: Garrett Brown <garrett.brown@aclima.io>
Date: Tue, 13 Oct 2020 03:51:36 -0700
Subject: [PATCH] Fix build error due to commonjs translation bug

Error was:

    Duplicate export '__esModule' (4:185) in snowpack-wrap:./node_modules/p2p-media-loader-hlsjs/dist/index.js
---
 build/p2p-media-loader-hlsjs.js | 6 ------
 dist/engine.js                  | 1 -
 dist/hlsjs-loader.js            | 1 -
 dist/index.js                   | 1 -
 dist/segment-manager.js         | 1 -
 5 files changed, 10 deletions(-)

diff --git a/build/p2p-media-loader-hlsjs.js b/build/p2p-media-loader-hlsjs.js
index 1567365..58ee7c5 100644
--- a/build/p2p-media-loader-hlsjs.js
+++ b/build/p2p-media-loader-hlsjs.js
@@ -38,7 +38,6 @@ window.p2pml.hlsjs = require("p2p-media-loader-hlsjs");
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-Object.defineProperty(exports, "__esModule", { value: true });
 const events_1 = require("events");
 const p2p_media_loader_core_1 = require("p2p-media-loader-core");
 const segment_manager_1 = require("./segment-manager");
@@ -146,7 +145,6 @@ module.exports.createHlsJsLoaderClass = createHlsJsLoaderClass;
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-Object.defineProperty(exports, "__esModule", { value: true });
 const DEFAULT_DOWNLOAD_LATENCY = 1;
 const DEFAULT_DOWNLOAD_BANDWIDTH = 12500; // bytes per millisecond
 class HlsJsLoader {
@@ -232,7 +230,6 @@ exports.HlsJsLoader = HlsJsLoader;
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-Object.defineProperty(exports, "__esModule", { value: true });
 const p2p_media_loader_core_1 = require("p2p-media-loader-core");
 const m3u8_parser_1 = require("m3u8-parser");
 const defaultSettings = {
@@ -612,8 +609,6 @@ module.exports = win;
 /*! @name m3u8-parser @version 4.4.0 @license Apache-2.0 */
 'use strict';
 
-Object.defineProperty(exports, '__esModule', { value: true });
-
 function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }
 
 var window = _interopDefault(require('global/window'));
@@ -1963,7 +1958,6 @@ exports.Parser = Parser;
 function __export(m) {
     for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
 }
-Object.defineProperty(exports, "__esModule", { value: true });
 exports.version = "0.6.2";
 __export(require("./engine"));
 __export(require("./segment-manager"));
diff --git a/dist/engine.js b/dist/engine.js
index 3ea119c..5a55179 100644
--- a/dist/engine.js
+++ b/dist/engine.js
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-Object.defineProperty(exports, "__esModule", { value: true });
 const events_1 = require("events");
 const p2p_media_loader_core_1 = require("p2p-media-loader-core");
 const segment_manager_1 = require("./segment-manager");
diff --git a/dist/hlsjs-loader.js b/dist/hlsjs-loader.js
index 8a81f2e..9093e62 100644
--- a/dist/hlsjs-loader.js
+++ b/dist/hlsjs-loader.js
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-Object.defineProperty(exports, "__esModule", { value: true });
 const DEFAULT_DOWNLOAD_LATENCY = 1;
 const DEFAULT_DOWNLOAD_BANDWIDTH = 12500; // bytes per millisecond
 class HlsJsLoader {
diff --git a/dist/index.js b/dist/index.js
index b863bea..9554078 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -18,7 +18,6 @@
 function __export(m) {
     for (var p in m) if (!exports.hasOwnProperty(p)) exports[p] = m[p];
 }
-Object.defineProperty(exports, "__esModule", { value: true });
 exports.version = "0.6.2";
 __export(require("./engine"));
 __export(require("./segment-manager"));
diff --git a/dist/segment-manager.js b/dist/segment-manager.js
index 325e356..7419b1a 100644
--- a/dist/segment-manager.js
+++ b/dist/segment-manager.js
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-Object.defineProperty(exports, "__esModule", { value: true });
 const p2p_media_loader_core_1 = require("p2p-media-loader-core");
 const m3u8_parser_1 = require("m3u8-parser");
 const defaultSettings = {
-- 
2.17.1

